pkgs/applications/networking/pjsip/default.nix

```diff
@@ -41,21 +45,28 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ openssl libsamplerate ]
     ++ lib.optional stdenv.isLinux alsa-lib
-    ++ lib.optional stdenv.isDarwin AppKit;
+    ++ lib.optionals stdenv.isDarwin [ AppKit CoreFoundation Security ];
 
-  preConfigure = ''
-    export LD=$CC
-  '';
+
+  NIX_CFLAGS_LINK = lib.optionalString stdenv.isDarwin
+                      "-headerpad_max_install_names";
 
   postBuild = lib.optionalString pythonSupport ''
     make -C pjsip-apps/src/swig/python
   '';
 
+  preConfigure = ''
+    export LD=$CC
+  '';
+
+  configureFlags = [ "--enable-shared" ]
+    # darwin-ssl is a deprecated SSL library that requires
+    # a more recent version of Apple SDK 10 which is not in nixpkgs.
```

Comment:
That is wrong, see https://github.com/NixOS/nixpkgs/blob/54b3a83b4483d34daec601256769889bb11e0e81/doc/stdenv/platform-notes.chapter.md?plain=1#L66 how to use the newer sdk

Reply:
Thanks. That was the missing link. Works great! :smile: